src/sb_pointcloud_processing/src/DBSCAN.cpp

```diff
@@ -83,6 +80,9 @@ void DBSCAN::expand(unsigned int center_index,
 }
 
 void DBSCAN::findNeighbors() {
+    this->_neighbors = new vector<unsigned int>[this->_pcl.size()];
+
+#pragma omp parallel for if (this->_pcl.size() > SEQUENTIAL_CUT_OFF)
```

Comment:
Glad to see this worked! This is really easy to read now. A quick comment explaining what this line is doing would be good for people who've never worked with pre-compiler directives / openmp before (I'm also a tad confused on how this `if` option works, I've never used it before).

Reply:
Just another note, out of curiosity, did we see any performance change after the switch to openmp?

Reply:
They were very negligible, if any.